*** /home/malabarth/carbon/system/netd/CommandListener.cpp	2014-05-28 11:55:02.635110688 +0200
--- /home/malabarth/carbon.old/system/netd/CommandListener.cpp	2014-05-27 21:52:50.141251421 +0200
***************
*** 1064,1070 ****
          return 0;
      }
  
!     if (!strcmp(argv[1], "startap")) {
          rc = sSoftapCtrl->startSoftap();
      } else if (!strcmp(argv[1], "stopap")) {
          rc = sSoftapCtrl->stopSoftap();
--- 1064,1074 ----
          return 0;
      }
  
!     if (!strcmp(argv[1], "start")) {
!         rc = sSoftapCtrl->startDriver(argv[2]);
!     } else if (!strcmp(argv[1], "stop")) {
!         rc = sSoftapCtrl->stopDriver(argv[2]);
!     } else if (!strcmp(argv[1], "startap")) {
          rc = sSoftapCtrl->startSoftap();
      } else if (!strcmp(argv[1], "stopap")) {
          rc = sSoftapCtrl->stopSoftap();
