*** /home/malabarth/carbon/frameworks/base/core/java/android/os/INetworkManagementService.aidl	2014-05-28 11:46:50.305096322 +0200
--- /home/malabarth/carbon.old/frameworks/base/core/java/android/os/INetworkManagementService.aidl	2014-05-27 21:47:03.404574637 +0200
***************
*** 238,254 ****
      /**
       * Start Wifi Access Point
       */
!     void startAccessPoint(in WifiConfiguration wifiConfig, String iface);
  
      /**
       * Stop Wifi Access Point
       */
!     void stopAccessPoint(String iface);
  
      /**
       * Set Access Point config
       */
!     void setAccessPoint(in WifiConfiguration wifiConfig, String iface);
  
      /**
       ** DATA USAGE RELATED
--- 238,254 ----
      /**
       * Start Wifi Access Point
       */
!     void startAccessPoint(in WifiConfiguration wifiConfig, String wlanIface, String softapIface);
  
      /**
       * Stop Wifi Access Point
       */
!     void stopAccessPoint(String wlanIface);
  
      /**
       * Set Access Point config
       */
!     void setAccessPoint(in WifiConfiguration wifiConfig, String wlanIface, String softapIface);
  
      /**
       ** DATA USAGE RELATED
