*** /home/malabarth/carbon/system/netd/SoftapController.h	2014-05-28 11:55:02.638444022 +0200
--- /home/malabarth/carbon.old/system/netd/SoftapController.h	2014-05-27 22:06:08.894608062 +0200
***************
*** 32,44 ****
--- 32,52 ----
      SoftapController();
      virtual ~SoftapController();
  
+     int startDriver(char *iface);
+     int stopDriver(char *iface);
      int startSoftap();
      int stopSoftap();
      bool isSoftapStarted();
      int setSoftap(int argc, char *argv[]);
      int fwReloadSoftap(int argc, char *argv[]);
  private:
+     char mBuf[SOFTAP_MAX_BUFFER_SIZE];
+     char mIface[IFNAMSIZ];
      pid_t mPid;
+     int mSock;
+ 
+     int addParam(int pos, const char *cmd, const char *arg);
+     int setCommand(char *iface, const char *fname, unsigned buflen=0);
      void generatePsk(char *ssid, char *passphrase, char *psk);
  };
  
